### PR TITLE
📝 Write a blog post calling for 2023 venues

### PR DESCRIPTION
**Pull request by drewbrew**
Mostly copied and pasted with a few changes:

1. 2017 -> 2023
2. Philadelphia -> San Diego
3. Added an explicit note about gender-neutral restrooms
4. Added a list entry about multi-year contracts
5. Dropped the third track room

### CODE REVIEW

File: _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+author_url : /about
+author_avatar: defna
+show_avatar : true
+read_time : ""
+feature_image: 2016-06-08
+show_related_posts: false
+permalink: /announcements/2022/5/16/djangocon-us-call-for-venue-proposal-2023/
+---
+
+We’re in the middle of planning DjangoCon US 2022, so It’s time to think about 2023! Where will we be next year? We need your help in making that decision!
```
Comment (logankilpatrick) [inline _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md]:
```suggestion
We’re in the middle of planning DjangoCon US 2022, so it’s time to think about 2023! Where will we be next year? We need your help in making that decision!
```

File: _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+So you want your city to host DjangoCon US! Hurrah! We’re delighted by your interest. Here’s what you need to know, and what we need to know, to consider your proposal.
```
Comment (logankilpatrick) [inline _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md]:
```suggestion
So you want your city to host DjangoCon US? Hurrah! We’re delighted by your interest. Here’s what you need to know, and what we need to know, to consider your proposal.
```

File: _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+	  * Tuesday Main Conf - 100
+	  * Wednesday Main Conf - 80
+	  * Thursday Sprints - 30
+	  * Friday Sprints - 15
+
+	* The block should include the guaranteed ability to add 20% more rooms for any given night if needed without moving to an overflow hotel.
+	* Concessions policy (comp room nights per actualized room nights)
+	* A sample contract showing the terms of booking the block of rooms. Again, do not sign any contracts. DEFNA will handle the business side of things. We just need to see the terms the hotel proposes for the group block.
+
+5. Provide estimates on costs associated with flights, accommodation, meals, ground transportation, etc. for an overview of total cost for attendees.
```
Comment (logankilpatrick) [inline _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md]: Should we add examples here like costs for people flying in from major cities like LA, Chicago, NY, etc ?

**Comment by logankilpatrick** [comment]
Overall looks good after those points above are addressed

**Review by logankilpatrick — APPROVED** [review]

File: _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md (hunk 3 above)
Comment (drewbrew) [reply _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md]: Sure, that sounds good to me.

Comment (drewbrew) [reply _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md]: How about something like:

> Suggested cost examples include typical round-trip airfare from several major hub cities, including New York, Los Angeles, San Francisco, Atlanta, Dallas, Chicago, and London.

Comment (logankilpatrick) [reply _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md]: Yes, looks good!

Comment (logankilpatrick) [inline _posts/2022-05-16-djangocon-us-call-for-venue-proposal-2023.md]:
```suggestion
5. Provide estimates on costs associated with flights, accommodation, meals, ground transportation, etc. for an overview of the total cost for attendees.

   * Suggested cost examples include typical round-trip airfare from several major hub cities, including New York, Los Angeles, San Francisco, Atlanta, Dallas, Chicago, and London.
```

**Pull request merged**